roles/ansible/tower/manage-credential-types/filter_plugins/list-dict-diff.py

```diff
@@ -0,0 +1,16 @@
+def get_remaining_items(list_a, list_b, key_a, key_b):
```

Comment:
Rather than copying these files in here, just use symlinks - see https://github.com/redhat-cop/infra-ansible/tree/master/roles/ansible/tower/manage-credentials for an example (both `filter_plugins` and `library` )

Reply:
Done